otter/test/convergence/test_service.py

```diff
@@ -817,7 +823,8 @@ def setUp(self):
                    links=freeze([{'href': 'link2', 'rel': 'self'}]))
         )
         self.state_active = {}
-        self.cache = [thaw(self.servers[0].json), thaw(self.servers[1].json)]
+        self.cache = [thaw(self.servers[0].json.set('_is_as_active', True)),
+                      thaw(self.servers[1].json.set('_is_as_active', True))]
```

Comment:
And going with the first option requires `self.cache` to be changed because now the servers are actually on all of the load balancers they want to be, so `_is_as_active` should be set when they get cached.